pkg/kubectl/cmd/top/top_pod.go

```diff
@@ -105,6 +107,8 @@ func NewCmdTopPod(f cmdutil.Factory, o *TopPodOptions, streams genericclioptions
 	cmd.Flags().BoolVar(&o.PrintContainers, "containers", o.PrintContainers, "If present, print usage of containers within a pod.")
 	cmd.Flags().BoolVarP(&o.AllNamespaces, "all-namespaces", "A", o.AllNamespaces, "If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
 	cmd.Flags().BoolVar(&o.NoHeaders, "no-headers", o.NoHeaders, "If present, print output without headers.")
+	cmd.Flags().StringVar(&o.SortBy, "sort-by", o.SortBy, "Sort the output by columns, supports 'cpu', 'memory'")
```

Comment:
at this point it might be better to re-use the message and list of flags from a single location, instead of re-defining in multiple locations.